src/read.c

```diff
@@ -1268,15 +1268,14 @@ struct obj *sobj; /* scroll, or fake spellbook object for scroll-like spell */
     case SPE_REMOVE_CURSE: {
         register struct obj *obj;
 
-        You_feel(!Hallucination
-                     ? (!confused ? "like someone is helping you."
-                                  : "like you need some help.")
-                     : (!confused ? "in touch with the Universal Oneness."
-                                  : "the power of the Force against you!"));
-
```

Comment:
I can understand the wish to change something there. But I'd rather
have a special effect when cursed from which a new message would
logically follow instead  of just having the extended message only appear when
non-cursed.

Reply:
I'm totally OK with adding a new cursed effect to this scroll rather than just modifying its message. I don't think I ever came up with one, though, which is why it's not in this pull request.